src/module/actor/character/sheet.ts

```diff
@@ -1159,6 +1159,15 @@ class CharacterSheetPF2e<TActor extends CharacterPF2e> extends CreatureSheetPF2e
         return handlers;
     }
 
+    protected override _onChangeTab(event: MouseEvent, tabs: Tabs, active: string) {
+        super._onChangeTab(event, tabs, active);
+        for (const tab of (tabs["_nav"]?.querySelectorAll("[role=tab]") ?? [])) {
+            if (tab instanceof HTMLElement) {
+                tab.setAttribute("aria-selected", String(tab.dataset.tab === active));
+            }
+        }
+    }
+
```

Comment:
This might be worth moving to one of the base classes for our sheets.

Reply:
Yeah, go ahead and move it over

Reply:
I'm experimenting with a better selector as well.